File: app/src/main/java/seigneur/gauvain/mycourt/ui/base/BaseViewHolder.java
package seigneur.gauvain.mycourt.ui.base;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import butterknife.ButterKnife;

/**
 * Base view holder for RecyclerView. Allows to bind views easily
 */
public class BaseViewHolder extends RecyclerView.ViewHolder {

    public BaseViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/seigneur/gauvain/mycourt/ui/base/BaseViewHolder.kt
package seigneur.gauvain.mycourt.ui.base

import androidx.recyclerview.widget.RecyclerView
import android.view.View

import butterknife.ButterKnife

/**
 * Base view holder for RecyclerView. Allows to bind views easily
 */
open class BaseViewHolder(itemView: View) : androidx.recyclerview.widget.RecyclerView.ViewHolder(itemView) {

    init {
        ButterKnife.bind(this, itemView)
    }
}
